src/components/application_manager/test/application_helper_test.cc

```diff
@@ -243,7 +250,11 @@ TEST_F(ApplicationHelperTest, RecallApplicationData_ExpectHMICleanupRequests) {
   app_impl_->AddChoiceSet(choice_set_id, cmd[strings::msg_params]);
   app_impl_->SubscribeToButton(mobile_apis::ButtonName::AC);
 
-  EXPECT_CALL(*mock_message_helper_, SendUnsubscribedWayPoints(_));
+  smart_objects::SmartObjectSPtr message_to_hmi;
+  EXPECT_CALL(*mock_message_helper_,
```

Comment:
You check that message will be created. But I do not see check that message will be sent

Reply:
@LuxoftAKutsan I'm not sure that I can do this, because in this test real, not mock, instances of Application Manager and RPC service are used. That's why I cannot check this calls, or you can suggest some other ways of perform this check?